src/data/v2/stream.rs

```diff
@@ -44,6 +45,7 @@ use websocket_util::subscribe::MessageStream;
 use websocket_util::tungstenite::Error as WebSocketError;
 use websocket_util::wrap;
 use websocket_util::wrap::Wrapper;
+use crate::api::v2::asset::Exchange;
```

Comment:
Data and "API" are treated differently and we do not share publicly accessible data types between the two. Until Alpaca clarifies their relationship I don't intend to change that. I don't think I want to include the exchange here yet. Let's leave this change out for now.

Reply:
ok, changed that to just char

Reply:
> ok, changed that to just char

Thanks! I am not sure a `char` is appropriate, to be honest. This crate is about providing a convenient interface to the upstream functionality. Let's leave this out for now. If you need it, I suggest we open a new issue and/or pull request and we can discuss. I am not terribly opposed to sharing data structure between `api` and `data`, but we may want to double check with Alpaca how independent they are.